src/features/collectables/HotspotList.tsx

```diff
@@ -3,22 +3,23 @@ import { times } from 'lodash'
 import { FlatList } from 'react-native-gesture-handler'
 import { useSafeAreaInsets } from 'react-native-safe-area-context'
 import { RefreshControl } from 'react-native'
-import { useNavigation } from '@react-navigation/native'
+import { useIsFocused, useNavigation } from '@react-navigation/native'
 import { useTranslation } from 'react-i18next'
 import { useColors } from '../../theme/themeHooks'
 import Box from '../../components/Box'
-import { CollectableSkeleton } from './NftListItem'
-import { Collectable } from '../../types/solana'
+import { NFTSkeleton } from './NftListItem'
+import { CompressedNFT } from '../../types/solana'
 import { CollectableNavigationProp } from './collectablesTypes'
 import HotspotListItem from './HotspotListItem'
 import ButtonPressable from '../../components/ButtonPressable'
 import useHotspots from '../../hooks/useHotspots'
-import FadeInOut from '../../components/FadeInOut'
+import CircleLoader from '../../components/CircleLoader'
 
 const HotspotList = () => {
   const { bottom } = useSafeAreaInsets()
   const navigation = useNavigation<CollectableNavigationProp>()
   const { t } = useTranslation()
+  const isFocused = useIsFocused()
```

Comment:
This is cool, didn't know it existed. We have had issues in the past with focus states when you background and foreground the app on the screen. Can you just give it a try to see if it behaves properly?

Reply:
Let me test 👍🏽

Reply:
Looks like its working as intended